firmware/config/engines/custom_engine.cpp

```diff
@@ -889,7 +889,17 @@ void detectBoardType() {
 void fuelBenchMode() {
     engineConfiguration->cranking.rpm = 12000;
     setFlatInjectorLag(0);
-    engineConfiguration->postCrankingFactor = 1;
+
+	static const float defaultPostCrankingCLTBins[] = {
```

Comment:
why same (or not same?) code three times? why in such odd places?

Reply:
so it should be just in firmware/config/engines/custom_engine.cpp, correct?

Reply:
* should never be copy-paste
* custom_engine is some specific configuration, default_cranking.cpp seems most reasonable

Reply:
done!